src/platform/site-wide/tests/sitemap/sitemap-helpers.js

```diff
@@ -64,7 +64,15 @@ function runTests(client, segment, only508List) {
       .waitForElementVisible('body', Timeouts.normal)
       .axeCheck(
         'document',
-        only508 ? { scope: url, rules: ['section508'] } : { scope: url },
+        only508
```

Comment:
If we're deleting all of the sitemap tests, do we still need this `sitemap-helpers.js` file?

Reply:
we do because the virutal regress tests apparently use the helpers in there